website/docs/components/form/primitives/partials/guidelines/overview.md

```diff
@@ -7,6 +7,8 @@ Form Primitives are used to compose form fields.
 - `Form::Legend` is the legend associated to the fieldset
 - `Form::Field` is the generic container for control, label, helper text and error messaging
 - `Form::Fieldset` is the generic container to group multiple fields with label, helper text, and error messaging
+- `Form::CharacterCount` is optionally used to display the number of characters entered in a field, the maximum number of characters allowed, or a custom message communicating the relationship between the count and the maximum length.
+    - `CharacterCount` is only used in fields that accept text values (TextInput, Textarea, MaskedInput, etc).
```

Comment:
I was not aware we were planning to introduce this change to the `MaskedInput` too – is there a use case for such scenario, or is it us trying to be consistent across form fields?

Reply:
This was resolved by adding the `CharacterCount` to `MaskedInput`